### PR TITLE
#167729782-docker-setup

### PR DESCRIPTION
**Pull request by okabamac**
## What does this PR do?
- Setup dockerfiles and dockercompose scripts

## Description of Task to be completed?
- Create dockerfile for building images
- Create dockercompose scripts to enable running docker containers easy
- Delete .env.sample file because of duplication
- Write scripts in package.json to make running commands easy

## How should this be manually tested?
pull or fetch this branch
Run the following commands after you cd into the directory
``` docker-compose -f docker-compose.dev.yml up --build ``` 
To run it inside dockers(ensure you have docker and docker-compose installed
Any background context you want to provide?
No

## What are the relevant pivotal tracker stories?
[#167729782](https://www.pivotaltracker.com/story/show/167729782)

* Screenshots (if appropriate)

Questions 
No

### CODE REVIEW

**Review by chibytez — APPROVED** [review]

**Review by okafor-chidimma — APPROVED** [review]
No merge conflicts...Approved for merge

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - POSSTGRES_USER=$POSTGRES_USER
+      - POSTGRES_PASSWORD=$POSTGRES_PASSWORD
+      - DB_HOST=postgres
+      - NODE_ENV=production
+    depends_on:
+      - postgres-prod
+    networks: 
+      - webnet
+networks: 
+  webnet:
```
Comment (davidfrank96) [inline docker-compose.prod.yml:27]: This red symbol means that after writing code ..you should always add space at the end... so try to implement that moving forward

**Review by davidfrank96 — APPROVED** [review]
Looks good bruh, just check out the little comment below

File: docker-compose.prod.yml (hunk 1 above)
Comment (okabamac) [reply docker-compose.prod.yml:27]: OK. Thanks a lot, I probably forgot to run lint at the end of the last commit.

**Review by Stylll — APPROVED** [review]

**Pull request merged**